docs/source/models/intervention_models/intrapartum/azithromycin_intervention.rst

```diff
@@ -0,0 +1,265 @@
+.. _azithromycin_intervention:
+
+=========================================
+Azithromycin for treating maternal sepsis
+=========================================
+
+.. contents::
+   :local:
+   :depth: 1
+
+.. list-table:: Abbreviations
+  :widths: 15 15 15
+  :header-rows: 1
+
+  * - Abbreviation
+    - Definition
+    - Note
+  * - BEmONC
+    - Basic emergency obstetric and neonatal care
+    - Operationalized as facilities without C-section capabilities
+  * - CEmONC
+    - Comprehensive emergency obstetric and neonatal care
+    - Operationalized as facilities with capabilities to perform  C-section
+  * - PAF
+    - Population Attributable Fraction
+    - 
+  * - RR
+    - Relative Risk
+    - 
+  * - SARA
+    - Services Availabiity and Readiness Assessment
+    - 
+
+Intervention Overview
+-----------------------
+
+Azithromycin is used to prevent maternal sepsis in the intrapartum period, reducing the risk incidence.
+
+This section describes how an azithromycin intervention can be implemented and calibrated for the :ref:`MNCNH Portfolio model <2024_concept_model_vivarium_mncnh_portfolio>`.
+
+.. list-table:: Affected Outcomes
+  :widths: 15 15 15 15
+  :header-rows: 1
+
+  * - Outcome
+    - Effect
+    - Modeled?
+    - Note (ex: is this relationship direct or mediated?)
+  * - Maternal sepsis and other maternal infections Incidence Probability :math:`\text{CSMR}_i^\text{sepsis}`
+    - Adjust multiplicatively using RR
+    - Yes
+    - For convenience, we will model this like a dichotomous risk factor; more details below
+
+Baseline Coverage Data
+++++++++++++++++++++++++
+
+Baseline coverage data for azithromycin in CEmONC and BEmONC is only reflective of Ethiopian health systems in 2015-2018. These 
+placeholder values come from three data sources, all for Ethiopia, all identified by the Health Systems team at IHME: the 2016 
+Ethiopia EmONC Final Report found 39.2% of BEmONC facilities and 90.4% of CEmONC facilities have azithromycin; the 2016 SARA Report 
+found 68.4% of BEmONC facilities and 90.5% of CEmONC facilities have azithromycin; the 2018 SARA Report found 75.9% of BEmONC 
+facilities and 98.0% of CEmONC facilities have azithromycin. While we plan a data strategy to fill the gaps we have used a simple 
+average.
+
+.. list-table:: Baseline Coverage of intrapartum azithromycin
```

Comment:
Hmm, I seem to remember that routine use of prophylactic azithromycin during labor (for all pregnancies) is not standard practice but rather something new that GF wanted to advocate to become the new standard of care... did you see anything about this in your searching? It may change our strategy for how to inform this. 

If you haven't seen anything about it we should check with the gates foundation at our upcoming meeting!

Reply:
Ok yeah I found this paper in a quick search and it looks like they are reporting that prophylactic intrampratum antibiotics in African sites is basically zero. There was some coverage for Pakistan though! https://obgyn-onlinelibrary-wiley-com.offcampus.lib.washington.edu/doi/10.1111/1471-0528.17930

Reply:
So basically I think assuming that hospitals that have azithromycin are using it prophylactically in labor is too optimistic!